src/service-api.ts

```diff
@@ -151,6 +152,15 @@ const plugin: FastifyPluginAsync<GraaspChatPluginOptions> = async (
       },
     );
 
+    fastify.get<{ Params: { itemId: string } }>(
+      '/:itemId/export/chat',
```

Comment:
What about `/chat/export`?

Reply:
@pyphilia This is what i wanted to use, but it seemed like it got confused with the other `GET` route. So i don't know if i did something wrong or something else ...